collect_app/src/main/res/values/strings.xml

```diff
@@ -352,8 +352,9 @@
     <string name="polygon_validator">Must have at least 3 points to create Polygon</string>
     <string name="polyline_validator">Must have at least 2 points to create Polyline</string>
     <string name="clear">Clear</string>
-    <string name="manual_mode">Manual recording</string>
-    <string name="automatic_mode">Automatic recording</string>
+    <string name="placement_mode">Tap to place points</string>
+    <string name="manual_mode">Manual GPS recording</string>
```

Comment:
I don't think the naming is accurate since it's not technically GPS. How does this feel?

- Manual - Tap map to place points
- Manual - Use location provider
- Automatic - Use location provider

Reply:
It doesn't feel as good as the current wording to me.  "Manual" is slightly confusing because in the first item the *time and position* are manually controlled, whereas in the second item the *time* is manually controlled but the position is recorded from a sensor.  Also, I'm not sure that "provider" makes sense to users—I don't think I've seen that word in any location-related UI; in Settings, the choice of provider is called "Location mode".

When writing this PR, I tried to come up with terminology for this that I could use consistently in the UI and in our code; here's what I was going for:
  - Clearly different names for each of the three ways of entering points
  - Ideally a single word that characterizes each option
  - Putting the distinguishing word at the beginning of the option
  - Phrases that aren't too different from the current text ("Manual recording", "Automatic recording") or the previous text ("Manual Mode", "Automatic Mode"), so they're recognizable to existing users
  - Terminology that separates the first option (location from a tap) and groups the other two (location from the sensor)

To elaborate on the last criterion, I think the similarity hierarchy should look like this:

  - Let the user indicate the position of each point
      - A. Tap the map to place points
  - Read the location from a sensor
      - B. Read from a sensor on user command
      - C. Read from a sensor at regular intervals

and not like this:

  - Place each point on user command
      - A. Place the point according to where the user tapped
      - B. Place the point according to a sensor reading
  - Place points automatically at regular intervals
      - C. Place points at regular intervals according to the sensor reading

because the use case for A is quite different from the use cases for B and C.

Thus I need a term for (A or B or C), a term for A, a term for (B or C), a term for B, and a term for C.  I went with:

  - Data entry
      - Placement
          - Tap to place points
      - Recording
          - Manual GPS recording
          - Automatic GPS recording

The code uses the same terminology (placement, recording mode, automatic recording, etc.) except that the topmost term in the code is "input" rather than "data entry". The choice of "placement" is inspired by the use of "placement-map" to have a similar meaning in the existing GeoPoint activity, where "placement" means "placed by the user".

I see what you mean about "GPS" being potentially wrong.  Instead of "data entry", how about "input method"?  What if we used:

  - Input method
      - Placement
          - Placement by tapping
      - Recording
          - Manual location recording
          - Automatic location recording

Is that better?

Reply:
Thanks for the explanation, Ping. Naming is hard. I think Input method, placement by tapping, manual/automatic location recording is fine. I'd like to request feedback from @cooperka and @smoyth, but we shouldn't block testing and merging because of that.

Reply:
Thanks!  I've made these changes for now.

Reply:
I rather liked the original options, but with "input method":

```
Input method:
- Tap to place points
- Manual GPS recording
- Automatic GPS recording
```

They sound explicit and clear. The new options (below for clarity)

```
Input method:
- Placement by tapping
- Manual location recording
- Automatic location recording
```

these to me sound more ambiguous. I understand GPS is *technically* only one of several location provider methodologies, but I think most users don't know the difference and even those who do would understand that it means whatever location provider is on the device.

I like "GPS" because it makes it clear that there's technology being used to determine your location, as opposed to "Manual location recording" which sounds like I might be able to type in my current address or something else entirely.

Perhaps instead of "GPS" we could say "position"? Not to throw a wrench in things, but what about:

```
Input method:
- Tap to place points
- Manually record positions
- Automatically record positions
```

[I don't want this to prevent merging, feel free to punt to a separate issue.]

Reply:
Hahaha, yes, you've independently hit upon the reason I initially went with "GPS".

I do see what you're trying to achieve, but "Manually record positions" actually sounds even less like using sensor readings, to me.  I think it's because "location" is the word Android has standardized upon to mean "the sensed location of your phone"; I don't see "position" in any of the standard UI.  Also, somehow, the imperative "record X" sounds more like the user is doing it than "X recording".

But, since we all have slightly different impressions, that probably means none of the wordings so far is truly strong enough to be obvious to everyone.

I'm open trying to find something even better, though.

Reply:
One more try... perhaps "your" will help?

    Input method
      - Place points manually
      - Manually record your location
      - Automatically record your location

Feel free to make other suggestions!

Reply:
I like your latest suggestion as-is! 👍